Sources/ArcGISToolkit/Components/Offline/OfflineMapInfo.swift

```diff
@@ -62,12 +62,13 @@ public extension OfflineMapInfo {
     var description: String { info.description }
     /// The URL of the portal item associated with the map.
     var portalItemURL: URL { info.portalItemURL }
+}
+
+extension OfflineMapInfo: Identifiable {
     /// The ID of the portal item associated with the map.
-    var portalItemID: Item.ID { .init(info.portalItemID)! }
+    public var id: Item.ID { .init(info.portalItemID)! }
 }
 
-private extension OfflineMapInfo {
-    /// The codable info is stored in json.
```

Comment:
Please revert.

Reply:
I don't understand how this ended up like that.